cli/src/common/boto3/cfn.py

```diff
@@ -20,8 +20,18 @@ def __init__(self):
         super().__init__("cloudformation")
 
     @AWSExceptionHandler.handle_client_exception
-    def create_stack(self, stack_name: str, template_url: str, disable_rollback: bool, tags: list):
+    def create_stack(
+        self, stack_name: str, disable_rollback: bool, tags: list, template_url: str = None, template_body: str = None
+    ):
         """Create CFN stack by using the given template."""
+        if template_url is None:
```

Comment:
let's add a check to be sure that at least one from template_body and template_url is set. or we can create two different utilities:
* create_stack_from_url
* create_stack

Reply:
Done, create two funcs.